### ISSUE TITLE
C++ extractor doesn't work on Centos 7

### ISSUE THREAD

**Issue by Cornelius-Riemenschneider** [github/codeql-cli-binaries#22]
Hi,

on Centos 7.7.1908, the cpp extractor fails to run:
```
/home/qldev/build/codeql/cpp/tools/linux64/extractor: /lib64/libstdc++.so.6: version `CXXABI_1.3.9' not found (required by /home/qldev/build/codeql/cpp/tools/linux64/extractor)
```
Presumably that could be fixed by compiling it on an older system, or by statically linking in the C++ runtime?

The whole reason for me to run the CodeQL tooling on a Centos system is that my laptop doesn't have a lot of RAM, but VSCode allows it very easily to run an extension host on a remote system.
Thus, I just run the whole CodeQL environment on a server with more RAM and separate from my desktop system.

Let me know if you need more information.

**Comment by hmakholm** [comment]
Thanks -- I've passed the issue on to the C++ extractor people.

**Comment by igfoo** [comment]
Thanks for the report! This should now be fixed in master, although I don't have a Centos machine to test it on.

**Comment by Cornelius-Riemenschneider** [comment]
Thank you, this is indeed fixed with release 2.1.1.
